README.md

```diff
@@ -24,3 +24,52 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+
+# CUJ recording
+
+## Usage
+
+### executeCuj
+
+To get data on the latency of a CUJ running on a connected device, run 
+```bash
+./executeCuj <preparatory_actions> <measured_actions> <num_iterations> <r?>
+```
+where preparatory_actions consists of actions to be performed prior to the CUJ 
+in question and measured actions consists of the CUJ in question plus a final 
+"termination" action, which will only be able to be performed once the last
+action in the CUJ has fully terminated. num_iterations denotes the number of 
+iterations you'd like the test to run through your CUJ. Optionally, you may 
+follow the command with "r" to signify that you'd like the run of median length
+to be outputted by the program. (Note that if you'd like to record, the entire
+test, including eachiteration, much have length <= 3 minutes.)
+
+the action list arguments must be formatted as follows:
+
+```bash
+'{<action1>, <action2>, ... , <actionN>}'
+```
+and each action is either to launch an app, click on a particular View or to enter 
+text in a particular textbox. Actions are represented in one of the following forms:
+     
+      1. “start;package_name" to launch the specified package.
+      2. “edit;text_displayed;text_entered” to enter text_entered into a textbox, where text_displayed is the text currently visible in the textbox.
+      3. “click;text_displayed” to click a non-editable view (normally a text-view or button) on the screen which has text containing the string text_displayed displayed on it
+      4. “clickImage;text_description” to click a view without text (normally an image view) where the description of the view contains the string description_text
+     
+Note that all text fields are case sensitive. Additionally, if an action of form 
+2, 3, or 4 is followed by “;strict” the search for a corresponding view will enforce 
+that an exact match is found for text_displayed (in cases 2 or 3) or text_description 
+(in case 4).
+
+### stitch
+
+to stitch together several videos to compare them side-by-side, use the stitch command 
+as follows:
+
+```bash
+./stitch <video1> <video2> ... <videoN>
```

Comment:
1. is there a command that records AND stitch?

2. what about `action_time_stamps.txt`?  how will that be incorporated?

Reply:
action_time_stamps.txt is cated to the screen after test is over